platforms/gke/base/_shared_config/workloads_variables.tf

```diff
@@ -21,5 +21,7 @@ locals {
 }
 
 variable "kueue_version" {
-  default = "0.8.1"
+  default     = "0.8.1"
```

Comment:
Worth updating this at least to the latest `0.8` patch version? https://github.com/kubernetes-sigs/kueue/releases